### ISSUE TITLE
manage --rootdir argument is not documented

### ISSUE THREAD

**Issue by ahmetalpbalkan** [docker/swarm#1090]
The `--rootdir` argument below has no documentation. Looks confusing.

```
$ docker run swarm:0.4.0-rc1 manage -h
Usage: swarm manage [OPTIONS] <discovery>

Manage a docker cluster

Arguments:
   <discovery>    discovery service to use [$SWARM_DISCOVERY]
                   * token://<token>
                   * consul://<ip>/<path>
                   * etcd://<ip1>,<ip2>/<path>
                   * file://path/to/file
                   * zk://<ip1>,<ip2>/<path>
                   * <ip1>,<ip2>

Options:
   --rootdir "/.swarm"
   --strategy "spread"						placement strategy to use [spread, binpack, random]
   --filter, -f [--filter option --filter option]		filter to use [affinity, health, constraint, port, dependency]
   --host, -H [--host option --host option]			ip/socket to listen on [$SWARM_HOST]
   --replication						Enable Swarm manager replication
   --advertise, --addr 						Address of the swarm manager joining the cluster. Other swarm manager(s) MUST be able to reach the swarm manager at this address. [$SWARM_ADVERTISE]
   --tls							use TLS; implied by --tlsverify=true
   --tlscacert 							trust only remotes providing a certificate signed by the CA given here
[...]
```

**Comment by thaJeztah** [comment]
Interested in making a PR? :smile:

**Comment by ahmetalpbalkan** [comment]
I'm not sure what it is abou @thaJeztah  :)

**Comment by thaJeztah** [comment]
Ah, ok, apologies in that case, I thought you did :+1:

**Comment by aluzzardi** [comment]
Indeed :) I think we are going to deprecate it soon enough.

**Comment by MHBauer** [comment]
@aluzzardi this is fixed by #1205 and can be close.

**Comment by jimmyxian** [comment]
@MHBauer Thanks :+1:
